modules/windows-containers-release-notes-limitations.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/windows-containers-release-notes-#-x
+
+[id="windows-containers-release-notes-limitations"]
+= Known limitations
+
+Note the following limitations when working with Windows Container Support for Red Hat OpenShift:
+
+* The following {product-title} features are not supported:
+** OpenShift Serverless
+** Red Hat OpenShift Pipelines
+** Thanos User Workload Monitoring
+** Service Mesh
+** Cost Management
+** CodeReady (Containers and Workspaces)
+** Odo
+** Builds v2 or BuildConfig or s2i
+** Any operating System Red Hat ships
+
+* Windows Nodes do not support workloads created by using deployment configs. You can use a deployment or other method to deploy workloads.
+
+* The WMCO is not supported in clusters that use a cluster-wide proxy because the WMCO is not able to route traffic through the proxy connection for the workloads.
+
+* Kubernetes has identified the following link:https://kubernetes.io/docs/concepts/windows/intro/#limitations[node feature limitations] :
+** HugePages: not supported for Windows containers
+** Privileged containers: not supported for Windows containers. HostProcess Containers offer similar functionality.
+** TerminationGracePeriod: requires containerD
```

Comment:
We can remove this bullet given we are using containerd.

Reply:
@aravindhp Does this list apply to all versions of WMCO? I was thinking we might need this for versions earlier than 6.0.0??

Reply:
That is correct. BTW, it should be `containerd`. Upstream needs to be fixed.